deepchem/models/lightning/dc_lightning_module.py

```diff
@@ -0,0 +1,99 @@
+import torch
+
+try:
+  import pytorch_lightning as pl  # noqa
+  PYTORCH_LIGHTNING_IMPORT_FAILED = False
+except ImportError:
+  PYTORCH_LIGHTNING_IMPORT_FAILED = True
+
+
+class DCLightningModule(pl.LightningModule):
+  """DeepChem Lightning Module to be used with Lightning trainer.
+
+  Example code
```

Comment:
Make sure to match numpy doc style exactly or the code won't render in the docs

https://numpydoc.readthedocs.io/en/latest/format.html